providers/shared/components/bastion/id.ftl

```diff
@@ -103,6 +103,32 @@
             {
                 "Names" : "OSPatching",
                 "Children" : osPatchingChildConfiguration
+            },
+            {
+                "Names" : "Image",
+                "Description" : "Configures the source of the virtual machine image used for the instance",
+                "Children" : [
+                    {
+                        "Names" : "Source",
+                        "Description" : "Where to source the image id from - Reference: uses the Regions AMIs reference property to find the image",
+                        "Values" : [ "Reference" ]
+                    },
+                    {
+                        "Names" : "Reference",
```

Comment:
Should this be `Source:Reference` to align with `Source:git`, `Source:containerregistry` and `Source:url` used elsewhere for similar purposes?